app/src/main/res/layout-land/topic_practice_fragment.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.topic.practice.TopicPracticeViewModel" />
+  </data>
+  <androidx.recyclerview.widget.RecyclerView
+    android:id="@+id/topic_practice_skill_list"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:clipToPadding="false"
+    android:overScrollMode="never"
+    android:paddingStart="48dp"
+    android:paddingEnd="48dp"
+    android:paddingBottom="172dp"
+    android:scrollbars="none"
+    app:data="@{viewModel.topicPracticeSkillLiveData}"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent" />
+</layout>
```

Comment:
`Ctrl + Alt + L` missing in this file too.

Reply:
Done